commit 5036a3eef972394e0051f633ae82ad5a10355ff3
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2019-06-06T15:57:23-07:00 (4 months ago)
    
    OS-7826 prstat man page says -v and -m are different, but they aren't

diff --git a/usr/src/man/man1m/prstat.1m b/usr/src/man/man1m/prstat.1m
index 8ca1ffa70b..9ac2792517 100644
--- a/usr/src/man/man1m/prstat.1m
+++ b/usr/src/man/man1m/prstat.1m
@@ -1,11 +1,11 @@
 '\" te
 .\" Copyright (c) 2013 Gary Mills
 .\" Copyright (c) 2006, 2009 Sun Microsystems, Inc. All Rights Reserved.
-.\" Copyright (c) 2018, Joyent, Inc. All Rights Reserved.
+.\" Copyright (c) 2019, Joyent, Inc. All Rights Reserved.
 .\" The contents of this file are subject to the terms of the Common Development and Distribution License (the "License").  You may not use this file except in compliance with the License. You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE or http://www.opensolaris.org/os/licensing.
 .\"  See the License for the specific language governing permissions and limitations under the License. When distributing Covered Code, include this CDDL HEADER in each file and include the License file at usr/src/OPENSOLARIS.LICENSE.  If applicable, add the following below this CDDL HEADER, with
 .\" the fields enclosed by brackets "[]" replaced with your own identifying information: Portions Copyright [yyyy] [name of copyright owner]
-.TH PRSTAT 1M "September 1, 2018"
+.TH PRSTAT 1M "June 6, 2019"
 .SH NAME
 prstat \- report active process statistics
 .SH SYNOPSIS
@@ -161,10 +161,12 @@ Report statistics for each light-weight process (\fBLWP\fR). By default,
 .ad
 .sp .6
 .RS 4n
-Report microstate process accounting information. In addition to all fields
-listed in \fB-v\fR mode, this mode also includes the percentage of time the
-process has spent processing system traps, text page faults, data page faults,
-waiting for user locks and waiting for \fBCPU\fR (latency time).
+Report microstate process accounting information. This output format includes
+the percentage of time the process has spent in user mode, in system mode, and
+sleeping. It also includes the number of voluntary and involuntary context
+switches, system calls, the number of signals received, and the percentage of
+time the process has spent processing system traps, text page faults, data page
+faults, waiting for user locks and waiting for \fBCPU\fR (latency time).
 .RE
 
 .sp
@@ -357,11 +359,7 @@ Report only processes whose real user \fBID\fR is in the given list. Each user
 .ad
 .sp .6
 .RS 4n
-Report verbose process usage. This output format includes the percentage of
-time the process has spent in user mode, in system mode, and sleeping. It also
-includes the number of voluntary and involuntary context switches, system calls
-and the number of signals received. Statistics that are not reported are marked
-with the \fB-\fR sign.
+Same as \fB-m\fR, for backwards compatibility
 .RE
 
 .sp
